Adding classes to dummy days so they can be easily styled

ninty9notout opened the pull request:
Only one valid change, the rest are whitespace changes that removes trailing spaces.

`if (dummy) return '<div class="lightpick__day is-dummy"></div>';`

wakirin commented:
See on css styles:
```
.lightpick__day:not(.is-disabled):hover {
    background-size: contain;
    background-image: url(data:image/svg+xml;charset=utf-8,%3Csvg...);
}
```
Your changes will require to add `:not(is-dummy)` or you will see circle when cursor is hover on dummy div.

There is don't need to use extra class, you can use this selector for choose dummy div:
`.lightpick__days > div:not([class])`

ninty9notout commented:
Thanks for the info, I've updated that. There was also an issue with the JS that caused `NaN` on update so I've amended that also.

Updated to include `:not(.is-dummy)`.

The direct selector is okay, but it's generally a best practice to have classes on all elements so we can write neater CSS as per the [BEM principles](http://getbem.com/introduction/).
